### PR TITLE
Copy code from To_Do_list project

### PR DESCRIPTION
**Pull request by AleksandarIvezic**
### This is the copy of the project To_Do_list I created last week.  The goal was to create a simple "To-do list".
-  It lists the things that you need to do and allows you to mark them as complete. 
-  You can add a new task, edit and remove each listed task, or remove all tasks that are completed.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+import change from './status';
+import {
+  addTask, editTask, updateTask, removeTask, clearCompleted,
+} from './crud';
+
+let taskList = [];
+const input = document.getElementById('input');
+const push = document.getElementById('push');
+
+function saveList(list) {
```
Comment (mmatongo) [inline src/index.js:11]: Use arrow functions, they provide cleaner syntax and proper readable code.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  const taskList = JSON.stringify(list);
+  localStorage.setItem('taskList', taskList);
+}
+
+const tasks = document.getElementById('tasks');
+
+function populateList() {
```
Comment (mmatongo) [inline src/index.js]: Use arrow functions, they provide cleaner syntax and proper readable code.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+input.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    addTask(taskList, input.value);
+    input.value = '';
+    saveList(taskList);
+    populateList();
+  }
+});
+
+function loadList() {
```
Comment (mmatongo) [inline src/index.js]: Use arrow functions, they provide cleaner syntax and proper readable code.

File: src/index.js (hunk 1 above)
Comment (mmatongo) [inline src/index.js]: Use arrow functions, they provide cleaner syntax and proper readable code.

**Review by mmatongo — COMMENTED** [review]
Hey @ShinobiWarior,

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- Good ReadMe
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: src/index.js (hunk 1 above)
Comment (AleksandarIvezic) [reply src/index.js]: Thanks for the review. 👍🏼

**Comment by mmatongo** [comment]
## Hi @ShinobiWarior , 

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

### Congratulations! 🎉 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**